includes/html/api_functions.inc.php

```diff
@@ -1019,15 +1019,28 @@ function get_port_info(Illuminate\Http\Request $request)
 
 function search_ports(Illuminate\Http\Request $request)
 {
+    $field = $request->route('field');
     $search = $request->route('search');
-    $value = "%$search%";
+
+    if (isset($search)) {
```

Comment:
Looks good overall, but might as well put this part in the isset($search) check below.

Reply:
changed